pulp_container/app/models.py

```diff
@@ -98,6 +99,7 @@ class Manifest(Content):
     digest = models.TextField(db_index=True)
     schema_version = models.IntegerField()
     media_type = models.TextField(choices=MANIFEST_CHOICES)
+    data = models.TextField(default="")
```

Comment:
@ipanova no reason really, should I change it?

Reply:
Any reason why would you go with `None`? I guess these two are interchangeable in the current context. We do not have any convention for default values for text fields, do we? I would leave this default as it is since it serves its purpose only during the migration phase.